movo_common/movo_ros/src/movo/movo_pan_tilt.py

```diff
@@ -257,8 +257,8 @@ def _handle_rsp(self,data_bytes):
             self._jcs.actual.velocities = self.js.velocity
             self._jcs.actual.accelerations = [0.0,0.0]
             try:
-                self._jcs.error.positions = map(operator.sub,self._jcs.desired.positions,self._jcs.actual.positions)
-                self._jcs.error.velocities = map(operator.sub,self._jcs.desired.velocities,self._jcs.actual.velocities)
+                self._jcs.error.positions = list(map(operator.sub,self._jcs.desired.positions,self._jcs.actual.positions))
```

Comment:
another subtle bug around the map function whose behavior changed between python2 and 3. keep this in mind when ur debugging ur own tasks